designs/ClientDevProcess.md

```diff
@@ -0,0 +1,151 @@
+
+## Sprint Tracking
+
+Previously, the Client Team used GitHub labels to track its 3-week sprints, using the format `Sprint <number>`, and Milestones to track release versions. Now, the Client Team uses GitHub Milestones to track sprint tasks. This is done for one main reason: enabling the [Velocity Tracking](https://github.com/NuGet/Home#workspaces/nuget-client-team-55aec9a240305cf007585881/reports/velocity) and [Burndown](https://github.com/NuGet/Home#workspaces/nuget-client-team-55aec9a240305cf007585881/reports/burndown?milestoneId=4983201) reports.
+
+Issues start off without an assigned sprint. During Sprint Planning at the beginning of each Sprint, items in the Backlog should be assigned to an appropriate Sprint Milestone, as well as (generally) assigned to someone who will be working on it during that sprint.
+
+Over time, doing things this way will allow the Client Team to get a sense of its overall Velocity, which will help with future planning and resourcing.
+
+At the beginning of every sprint, issues that were partially worked on but not completed should be re-estimated to reflect the amount of work **left to be done**. While this means velocity data may be partially "lost", [it's a better choice for the sake of estimating the new sprint, which is what should matter more](https://www.scrum.org/index.php/forum/scrum-forum/29033/re-estimate-story-points-or-not).
+
+## Quality/Engineering Backlog Handling
+
+By using certain tags, the Client Team maintains ongoing quality and engineering backlogs, which represent things like bugs, engineering improvements, and tech debt.
+
+Every 6 months or so, the Client Team dedicates one sprint just to these backlog issues, with a full-team push in that direction.
+
+## Release Tracking
+
+As mentioned above, the Client Team used GitHub Milestones to track releases. Since those are now being used for Sprints (for the sake of ZenHub tracking reports), release tracking is moved to the [Release feature](https://github.com/NuGet/Home#workspaces/nuget-client-team-55aec9a240305cf007585881/reports/release?release=5e0e5fbd021f7aa0ec95db18) in ZenHub.
+
+When issues are scheduled, they should be assigned an appropriate Release, along with a Sprint.
+
+In order to generate release changelogs, the team should use [the ZenHub API](https://github.com/ZenHubIO/API#get-all-the-issues-for-a-release-report) -- possibly through [ZenHub.NET](https://github.com/AlexGhiondea/ZenHub.NET/blob/21cd562b30570594beb3842b28c372d66f87dcc6/src/ZenHubReleaseClient.cs#L84-L90).
+
+## Tracking Private/Sensitive Issues
+
+This document focuses on **public** issue tracking. Private issues, such as private design specs and security issues are handled through a private repo and tracked separately.
+
+## The Board
+
+- Link: https://github.com/NuGet/Home#workspaces/nuget-client-team-55aec9a240305cf007585881/board?repos=29996513
+

```

Comment:
Can we track PRs in the board?

Reply:
We could, but since every PR generally has an issue associated with it, I'm afraid we would just be double-tracking work? What do you think?